vermouth/pdb/pdb.py

```diff
@@ -66,7 +66,7 @@ def keyfunc(graph, node_idx):
             temp_factor = get_not_none(node, 'temp_factor', 0)
             element = get_not_none(node, 'element', '')
             charge = get_not_none(node, 'charge', 0)
-            if charge:
+            if charge and not omit_charges:
```

Comment:
Are you sure this is the way to do it? This still puts file format issues on the user. Instead it might make more sense to see whether the charge is within <error> of an integer.

Reply:
We want to write either all the charges or none of them. This means that we would have to check if all the charges are integer. Doing this check is not very difficult, however its output is not predictable for the final user that ends up with or without charges in a non obviously predictable way. Having a full switch has the benefit of letting the user in control.

Reply:
You could argue the user made that choice by choosing to write it as a pdb file, but ok